main.js

```diff
@@ -48,21 +47,138 @@ form.addEventListener('submit', (e) => {
+let projectDetails = document.createElement('div');
+let projectHeader = document.createElement('h2');
+let projectDecription = document.createElement('p');
+let projecttools = document.createElement('div');
+let toolsunoredered = document.createElement('ul');
+let list = document.createElement('li');
+let projectButton = document.createElement('button');
+
+document.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('project-image1').src = projectData[3].imag1;
+  document.getElementById('project-image2').src = projectData[3].image2;
+  document.getElementById('project-image3').src = projectData[3].image3;
+  document.getElementById('project-image4').src = projectData[3].image4;
+
+  let allbutton = document.querySelectorAll('.project-link');
+
+  allbutton.forEach((e) => e.addEventListener('click', () => {
+    modal.classList.toggle('card');
+  }));
+});
+
+projectSectionTitle.className = 'recent';
+underLineImage.className = 'line-divider2';
+projectCard.className = 'inner-element';
+
+underLineImage.src = 'img/Vector4.png';
+underLineImage.setAttribute('alt', 'Project Image');
+projectSectionTitle.innerText = 'MY RECENT PROJECT';
+projectSection.append(projectSectionTitle, underLineImage);
+
+const projectData = [
+  4,
+  {
+    projectTitle: '',
+    projectDescription: "A website created for a digital marketing agency to help improve sales and give information about company's product. This website also helps the client to contact the company for a business deal and showcases the company's different services and packages.",
+  },
+  {
+    firstLang: 'CSS',
+    secondLang: 'HTML',
+    thirdLang: 'JavaScript',
+    fourthLang: 'Ruby',
+  },
+  {
+    imag1: 'img/fola2.PNG',
+    image2: 'img/fola3.PNG',
+    image3: 'img/spike.PNG',
+    image4: 'img/fola2.PNG',
+  },
+];
+// eslint-disable-next-line no-plusplus
+for (let i = 1; i <= projectData[0]; i++) {
+  const projects = `<div class="proof proof-space">
+          <img id="project-image${i}" class="project-img" src="" alt="Project33" />
+          <div class="inner-element">
+            <h3>MARKETING AGENCY</h3>
+            <p>${projectData[1].projectDescription}</p>
+            <div class="tools">
+              <ul>
+                <li class="line">${projectData[2].firstLang}</li>
+                <li class="line">${projectData[2].secondLang}</li>
+                <li class="line">${projectData[2].thirdLang}</li>
+                <li>${projectData[2].fourthLang}</li>
+              </ul>
+            </div>
+            <button class="project-link btnsee btncheck">See Project</button>
+          </div>
+        </div>`;
+
+  projectSection.innerHTML += projects;
+}
 
-// // console.log(projectData.projectLanguages.thirdLang);
+// eslint-disable-next-line no-multi-assign
+const popupcard = document.getElementById('card');
+const popcard = `<div class="card-header">
+    <h2>Multi Post Story</h2>
+  <div class="class-close">&times;</div>
+  </div>
```

Comment:
- [ ] Kindly fix the x button. it should close the popup card when it's clicked.